dspace-server-webapp/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -2758,17 +2761,182 @@ public void testHiddenMetadataForUserWithWriteRights() throws Exception {
+        Collection collection = CollectionBuilder.createCollection(context, community).withName("Collection").build();
+        Item item = ItemBuilder.createItem(context, collection)
+            .withTitle("Publication1")
+            .withAuthor("Testy, TEst")
+            .withIssueDate("2015-01-01")
+            .withRelationshipType("Publication")
+            .build();
+        context.restoreAuthSystemState();
+
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        getClient(token).perform(get("/api/core/items/" + item.getID()))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$", ItemMatcher.matchItemProperties(item)))
+            .andExpect(jsonPath("$.entityType", is("Publication")));
+    }
+
+    @Test
+    public void testEntityTypeOrgUnit() throws Exception {
+        context.turnOffAuthorisationSystem();
+        EntityType orgUnit = EntityTypeBuilder.createEntityTypeBuilder(context, "OrgUnit").build();
+        Community community = CommunityBuilder.createCommunity(context)
+            .withName("Parent Community")
+            .build();
+        Collection collection = CollectionBuilder.createCollection(context, community).withName("Collection").build();
+        Item item = ItemBuilder.createItem(context, collection)
+            .withTitle("OrgUnit1")
+            .withAuthor("Testy, TEst")
+            .withIssueDate("2015-01-01")
+            .withRelationshipType("OrgUnit")
+            .build();
+        context.restoreAuthSystemState();
+
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        getClient(token).perform(get("/api/core/items/" + item.getID()))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$", ItemMatcher.matchItemProperties(item)))
+            .andExpect(jsonPath("$.entityType", is("OrgUnit")));
+    }
+
+    @Test
+    public void testEntityTypeProject() throws Exception {
+        context.turnOffAuthorisationSystem();
+        EntityType project = EntityTypeBuilder.createEntityTypeBuilder(context, "Project").build();
+        Community community = CommunityBuilder.createCommunity(context)
+            .withName("Parent Community")
+            .build();
+        Collection collection = CollectionBuilder.createCollection(context, community).withName("Collection").build();
+        Item item = ItemBuilder.createItem(context, collection)
+            .withTitle("Project1")
+            .withAuthor("Testy, TEst")
+            .withIssueDate("2015-01-01")
+            .withRelationshipType("Project")
+            .build();
+        context.restoreAuthSystemState();
+
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        getClient(token).perform(get("/api/core/items/" + item.getID()))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$", ItemMatcher.matchItemProperties(item)))
+            .andExpect(jsonPath("$.entityType", is("Project")));
+    }
```

Comment:
these tests seem to add no value compared to the first test testEntityTypePerson I would suggest to remove them as they are redundant. It would be useful instead to make test call also with the admin user and as anonymous

Reply:
I would prefer to keep at least tests for 2 different types in place, but we can drop the last 2

Admin and anonymous can be included as well